### ISSUE TITLE
nativefier --version fails for 7.6.6

### ISSUE THREAD

**Issue by ilovezfs** [jiahaog/nativefier#660]
```
iMac-TMP:~ joe$ ./node_modules/.bin/nativefier --version
TypeError [ERR_INVALID_ARG_TYPE]: The "url" argument must be of type string. Received type undefined
    at Url.parse (url.js:150:11)
    at Object.urlParse [as parse] (url.js:144:13)
    at appendProtocol (/Users/joe/node_modules/nativefier/lib/options/options/normalizeUrl.js:5:22)
    at normalizeUrl (/Users/joe/node_modules/nativefier/lib/options/options/normalizeUrl.js:13:27)
    at exports.default (/Users/joe/node_modules/nativefier/lib/options/options/optionsMain.js:20:16)
    at /Users/joe/node_modules/nativefier/lib/build/build/buildMain.js:175:9
    at nextTask (/Users/joe/node_modules/nativefier/node_modules/async/dist/async.js:5324:14)
    at Object.waterfall (/Users/joe/node_modules/nativefier/node_modules/async/dist/async.js:5334:5)
    at buildMain (/Users/joe/node_modules/nativefier/lib/build/build/buildMain.js:171:9)
    at Object.<anonymous> (/Users/joe/node_modules/nativefier/lib/cli.js:209:16)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
iMac-TMP:~ joe$ 
```

**Comment by ronjouch** [comment]
@ilovezfs fixed with 0c9ef088a534f15e916f5cde38fbdd37b71f92ce and [Nativefier 7.6.7](https://github.com/jiahaog/nativefier/releases/tag/v7.6.7), which should be [on npm](https://www.npmjs.com/package/nativefier) in a few minutes/hours.

**Comment by ilovezfs** [comment]
Thanks @ronjouch! 7.6.7 shipped in Homebrew: https://github.com/Homebrew/homebrew-core/pull/30664
